themes/custom/az_barrio/config/install/az_barrio.settings.yml

```
@@ -1,107 +1,136 @@
+langcode: en
+favicon:
+  use_default: true
+features:
+  comment_user_picture: true
+  comment_user_verification: true
+  favicon: true
+  node_user_picture: true
+logo:
+  path: profiles/custom/az_quickstart/themes/custom/az_barrio/images/logo.png
+  use_default: false
 bootstrap_barrio_library: production
+bootstrap_barrio_source: ''
+bootstrap_barrio_fluid_container: 0
+bootstrap_barrio_container: container
 bootstrap_barrio_region_clean_alert: false
+bootstrap_barrio_region_class_alert: ''
 bootstrap_barrio_region_clean_header_ua_utilities: false
-bootstrap_barrio_region_clean_branding: false
-bootstrap_barrio_region_clean_header: false
+bootstrap_barrio_region_class_header_ua_utilities: 'ms-auto d-none d-lg-block d-xl-block'
```

I think we don't need the last class since we already have `d-lg-block`:

```suggestion
bootstrap_barrio_region_class_header_ua_utilities: 'ms-auto d-none d-lg-block'
```